src/OpenTK.GraphicsLibraryFramework/Enums/ContextBoolAttributes.cs

```diff
@@ -0,0 +1,25 @@
+namespace OpenToolkit.GraphicsLibraryFramework
+{
+    /// <summary>
+    /// Context related boolean attributes.
+    /// </summary>
+    /// <seealso cref="IGLFW.WindowHint(ContextBoolAttributes,bool)"/>
+    public enum ContextBoolAttributes
+    {
+        /// <summary>
+        /// Is <c>true</c> if the window's context is an OpenGL forward-compatible one, or <c>false</c> otherwise.
```

Comment:
This makes it sound like you can use this to retrieve whether or not the context is forward-compatible; the exact opposite is true.

Reply:
I used https://www.glfw.org/docs/3.3/window_guide.html#window_attribs_ctx as a basis for these comments.

> GLFW_OPENGL_FORWARD_COMPAT is GLFW_TRUE if the window's context is an OpenGL forward-compatible one, or GLFW_FALSE otherwise.

Should I add it inside of a "remarks" tag?

Reply:
Maybe do something like

"Sets whether or not the window's context is an OpenGL forward-compatible one."?